WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -29,6 +29,9 @@ class CompetitionsController < ApplicationController
 
   def new
     @competition = Competition.new
+    if current_user.any_kind_of_delegate?
+      @competition.delegates |= [current_user]
```

Comment:
Are you sure that happens? I would have expected the fact that we're using ActiveRecord to prevent such a thing.

In this particular case, we just created a competition, so there are no delegates. Perhaps it would be cleanest to do:

```ruby
@competition.delegates = [current_user]
```

instead?